.eslintignore

```diff
@@ -0,0 +1,2 @@
+dist/**
+webpack.config.js
```

Comment:
@ggetz It needs to behave differently because it's a node script not a browser env and I missed the way to specify the override in eslint before. I just pushed a update to include it.